### ISSUE TITLE
Create new datasets

### ISSUE THREAD

**Issue by larek** [larek/mebel-tom-hanks#1]
## 1. How to start
1. Fork this [repository](https://github.com/larek/mebel-tom-hanks)
2. Clone to local machine
3. Create new branch

### Create new files in `src/data` directory with following filenames:
- mebel-chelny.ru.js
- mebel-naberejnye.ru.js
- lipeckayamebel.ru.js
- dkyalipeckamebel.ru.js
- krasnodarskayamebel.ru.js
- dlyakrasnodaramebel.ru.js
- voronejskayamebel.ru.js
- dlyavoronejamebel.ru.js
- volgogradskayamebel.ru.js
- dlyavolgogradamebel.ru.js
- ufimskayamebel.ru.js
- dlyaufymebel.ru.js
- rostovskayamebel.ru.js
- dlyarostovamebel.ru.js
- kurskayamebel.ru.js
- dlyakurskamebel.ru.js
- belgorodskayamebel.ru.js
- dlyabelgorodamebel.ru
- oskolskayamebel.ru.js
- dlyaoskolamebel.ru.js
- tambovskayamebel.ru.js
- dlyatambovamebel.ru.js
- dlyanijnegmebel.ru.js
- nijegorodkayamebel.ru.js

### Copy [this content](https://raw.githubusercontent.com/larek/mebel-tom-hanks/main/src/data/landing-4.js) to all files
in other words, you need to take the `src/data/landing-4.js` file, copy and rename it 24 times

### In each files update following fields:

- title
- description
- domain
- phone
- address

The necessary data can be found in this [link](https://docs.google.com/spreadsheets/d/1hU7h6QaXgB1lHrSYfGqbgRrGdGXYqOc-/edit?usp=sharing&ouid=100209127059039214455&rtpof=true&sd=true) (the problem is that the data on this link is structured like shit) 

## 3. Create a pull request
